pkgs/servers/mail/spf-engine/default.nix

```diff
@@ -1,12 +1,12 @@
 { lib, buildPythonApplication, fetchurl, pyspf }:
 
 buildPythonApplication rec {
-  pname = "pypolicyd-spf";
-  majorVersion = "2.0";
-  version = "${majorVersion}.2";
+  pname = "spf-engine";
+  majorVersion = builtins.head (builtins.match "([0-9]+\.[0-9]+)\..*" version);
+  version = "2.9.3";
 
   src = fetchurl {
-    url = "https://launchpad.net/pypolicyd-spf/${majorVersion}/${version}/+download/${pname}-${version}.tar.gz";
+    url = "https://launchpad.net/${pname}/${majorVersion}/${version}/+download/${pname}-${version}.tar.gz";
     sha256 = "1nm8y1jjgx6mxrbcxrbdnmkf8vglwp0wiw6jipzh641wb24gi76z";
```

Comment:
Hold up - the hash hasn't changed - we're still building the old package.

Reply:
Nice catch!